firedrake/slate/static_condensation/hybridization.py

```diff
@@ -199,17 +200,35 @@ def initialize(self, pc):
         # Make a SLATE tensor from Kform
         K = Tensor(Kform)
 
+        # Split forms
+        split_mixed_op = dict(split_form(Atilde.form))
+        id0, id1 = (self.vidx, self.pidx)
+        A00 = Tensor(split_mixed_op[(id0, id0)])
+        A01 = Tensor(split_mixed_op[(id0, id1)])
+        A10 = Tensor(split_mixed_op[(id1, id0)])
+        A11 = Tensor(split_mixed_op[(id1, id1)])
+        list_split_mixed_ops = [A00, A01, A10, A11]
+
+        split_trace_op = dict(split_form(K.form))
+        K0 = Tensor(split_trace_op[(0, id0)])
+        K1 = Tensor(split_trace_op[(0, id1)])
+        list_split_trace_ops = [K0, K1]
+
+        # Build schur complement operator and right hand side
+        nested = (prefix + "nested_schur" in PETSc.Options().getAll().keys()
+                  and PETSc.Options().getString(prefix + "nested_schur", "true") == 'true')
```

Comment:
I think `PETSc.Options(prefix).getBool("nested_schur", False)` is probably simpler.

Reply:
But maybe it doesn't work...?